test/distributed/org/apache/cassandra/distributed/test/thresholds/TombstoneCountWarningTest.java

```diff
@@ -68,10 +84,12 @@ public class TombstoneCountWarningTest extends TestBaseImpl
     @BeforeClass
     public static void setupClass() throws IOException
     {
+        logger.info("[test step : @BeforeClass] setupClass");
```

Comment:
these tests have very similar logs, so while debugging if we do not annotate test start its very hard to debug